src/components/viewStats/components/ViewStats.js

```diff
@@ -18,56 +21,122 @@ const countGender = (oldCount, person) => {
     return newCount;
 };
 
+// reducer function used to group the candidiates by age
+const countAge = (oldCount, person) => {
```

Comment:
only pass in `age` not the full person object into this function. Or if that is not possible desctructure age from the person object